README.md

```diff
@@ -50,9 +51,10 @@ this method should set the pizza's temperature to be 300. Note: you may not use
 the variable pizza inside your method.
 
 Your Answer:
-```js
-// write code here
-```
+var pizza = { temperature:70, type:cheese
+}
+this.bake(70+230)
+
 
```

Comment:
The `this` keyword only references `pizza` if it is inside of the `pizza` object. Also, you can set the temperature of 300 directly. Check out this example: https://github.com/ga-dc/wdi7-quiz2/tree/solution#question-4